schemas/network.json

```diff
@@ -0,0 +1,199 @@
+{
+  "$id": "https://fildev.network/schemas/network.json",
+  "type": "object",
+  "properties": {
+    "Name": {
+      "type": "string",
+      "examples": [
+        "calibration"
+      ]
+    },
+    "Purpose": {
+      "type": "string",
+      "examples": [
+        "For miners to prepare for the Space Race (recommended for most miners)"
+      ]
+    },
+    "Stability": {
+      "type": "string",
+      "enum": ["low", "moderate", "high"],
+      "examples": [
+        "moderate"
+      ]
+    },
+    "Status": {
+      "type": "string",
+      "enum": ["online", "offline", "unstable"],
+      "examples": [
+        "online"
+      ]
+    },
+    "NetworkParameters": {
+      "type": "object",
+      "properties": {
+        "SupportedProofTypes": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "Name": {
+                "type": "string",
+                "examples": [
+                  "RegisteredSealProof_StackedDrg512MiBV1"
+                ]
+              },
+              "Size": {
+                "type": "string",
```

Comment:
Should this be "integer"?

Reply:
It's defined as a string everywhere in the lotus API (as it's a BigInt), so for consistency (and easier generation) I'm keeping it as a string.